estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -95,29 +156,57 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
             this.apiService.apiProjectsSpidersRead(requestParams).then(
                 async (response: Spider) => {
                     const data = await this.getSpiderJobs(1);
+                    const completedJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "COMPLETED");
+                    const runningJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "RUNNING");
+                    const queueJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "IN_QUEUE");
+                    const tableStatus = [
+                        !(queueJobs.length === 0),
+                        !(runningJobs.length === 0),
+                        !(completedJobs.length === 0),
+                    ];
```

Comment:
Add error jobStatus, there are some jobs with error and that view doesn't show any table with that status

Reply:
Done adding additional table, and additional state parameter.